projecttemplates/fis-image-streams.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
Is this the standards image stream file for Fuse? Can't we call it from its original hosted location?

Reply:
Yes we can call it from original hosted location and removed from templates folder.